File: conexion_db_php/conexion_xamp.php
<?php
$host = "localhost";
$user = "root"; // Usuario por defecto en XAMPP
$password = ""; // Por defecto está vacío
$database = "tu_base_de_datos";

$conn = new mysqli($host, $user, $password, $database);

if ($conn->connect_error) {
    die("Error de conexión: " . $conn->connect_error);
}

echo "Conexión exitosa a MySQL en XAMPP";
?>

File: conexion_db_java/src/conexion_xamp.java
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;

public class MySQLConnection {
    public static void main(String[] args) {
        String url = "jdbc:mysql://localhost:3306/tu_base_de_datos";
        String user = "root";  // Cambia si tienes otro usuario
        String password = "";  // Deja vacío si no pusiste contraseña en XAMPP

        try (Connection conn = DriverManager.getConnection(url, user, password)) {
            System.out.println("Conexión exitosa a MySQL en XAMPP");
        } catch (SQLException e) {
            System.err.println("Error de conexión: " + e.getMessage());
        }
    }
}